pkg/apis/v1alpha1/workload.go

```diff
@@ -65,26 +71,63 @@ type WorkloadServiceClaimReference struct {
 }
 
 type WorkloadSpec struct {
-	Params []Param         `json:"params,omitempty"`
-	Source *Source         `json:"source,omitempty"`
-	Build  WorkloadBuild   `json:"build,omitempty"`
-	Env    []corev1.EnvVar `json:"env,omitempty"`
-	// Image is a pre-built image in a registry. It is an alternative to defining source
-	// code.
-	Image              *string                      `json:"image,omitempty"`
-	Resources          *corev1.ResourceRequirements `json:"resources,omitempty"`
-	ServiceAccountName string                       `json:"serviceAccountName,omitempty"`
-	ServiceClaims      []WorkloadServiceClaim       `json:"serviceClaims,omitempty"`
+	// Additional parameters.
+	// See: https://cartographer.sh/docs/latest/architecture/#parameter-hierarchy
+	// +optional
+	Params []OwnerParam `json:"params,omitempty"`
+
+	// The location of the source code for the workload. Specify
```

Comment:
I wonder if we could improve the verbiage here and also for `Image` down below so it's more congruent.

Reply:
Do you have a suggested edit?

Reply:
Yeah, probably would be more constructive. How about:

```
Image refers to a pre-built image in a registry. It is an alternative to specifying the location of source code for the workload. Specify one of `spec.source` or `spec.image`.
```

I toyed with the idea of making the "Specify one of..." language stronger, but trying to squeeze only/must or similar words in made me appreciate the extreme concision.